pkg/chartutil/capabilities.go

```diff
@@ -24,16 +27,21 @@ import (
 	helmversion "helm.sh/helm/v3/internal/version"
 )
 
+const (
+	k8sMajorVer = 1
+	k8sMinorVer = 19
```

Comment:
LDFLAGS uses the variable name `k8sVersionMajor` and `k8sVersionMinor`

Reply:
done